js/components/dataset/list.vue

```diff
@@ -21,7 +21,8 @@ export default {
                 key: 'title',
                 sort: 'title',
                 align: 'left',
-                type: 'text'
+                type: 'text',
+                width: 200
```

Comment:
1. Removing all widths gives this: 
![image](https://cloud.githubusercontent.com/assets/1301085/26588076/86619a8c-4554-11e7-910d-8847aa6e0bf1.png)
I feel better about not forcing sizes but I'm a bit bothered by the length of the third column, which was the initial issue.

2. Do you mean using classes for specific columns sizes? I agree this would be a better solution, this PR was meant as a quickfix but maybe we should rethink the datatable component then.